filesystem/src/main/java/org/frankframework/filesystem/LocalFileSystem.java

```diff
@@ -267,4 +276,12 @@ public String getRoot() {
 		return root;
 	}
 
+	/**
+	 * Whether the LocalFileSystem tries to create the root folder if it doesn't exist yet.
+	 * @ ff.default false
+	 * @param createRootFolder
+	 */
+	public void setCreateRootFolder(boolean createRootFolder) {
+		this.createRootFolder = createRootFolder;
+	}
```

Comment:
This property should be added to the `LocalFileSystemPipe` and `LocalFileSystemSender` and from there be copied to the LocalFileSystem, same as the `root` property.

Otherwise there is no way for users to configure this property in the Configuration.xml files.